lib/rust/ensogl/component/list-view/src/entry/list.rs

```diff
@@ -181,6 +192,16 @@ where E::Model: Default
         }
     }
 
+    /// Set params used in the displayed entries. Any entries newly created by the function will
+    /// use the styles located at the `style_prefix` path in the application's style sheet.
+    ///
+    /// The function should be used sparingly, as it may be a costly operation. The implementation
+    /// may recreate all the displayed entries from scratch.
+    pub fn set_entry_params(&self, params: E::Params, style_prefix: style::Path) {
```

Comment:
Instead of docs, this should be part of name, like `set_entry_params_and_recreate_entries`. This is then very visible in all usage-places.